### PR TITLE
Minor updates to the documentation

### PR DESCRIPTION
**Pull request by RemiLehe**
This adds 2 minor updates to the documentation:
- Fixes a typo in the install instructions for Cori
- Adds the moving window in the list of parameter (it seems that it was missing)

### CODE REVIEW

File: Docs/source/usage/parameters.rst
Comment (EZoni) [inline Docs/source/usage/parameters.rst]: Just a typo:
```suggestion
    (i.e. use ``1.0`` for a moving window that moves exactly at the speed of light)
```

**Review by EZoni — APPROVED** [review]
Thank you for updating the docs, Remi. I just left one minor comment.

**Comment by EZoni** [comment]
@RemiLehe I added a commit with the tag `[skip ci]` to fix a simple typo, aiming at a fast merge. However it did not skip the CI tests but rather the builds with the different compilers. As a result, I think I cannot merge this now, because some "statutes" are shown as still "required" or "expected". Maybe you can merge this anyways, if you have higher write/merge access.

**Comment by RemiLehe** [comment]
Sounds good. Thanks for letting me know.

**Pull request merged**
